docs/reference/ilm/ilm-tutorial.asciidoc

```diff
@@ -245,10 +247,11 @@ is met.
 --------------------------------------------------
 // TESTRESPONSE[skip:no way to know if we will get this response immediately]
 
-<1> The policy used to manage the index
-<2> The age of the index
-<3> The step {ilm-init} is performing on the index
-<4> The definition of the current phase (the `hot` phase)
+<1> The age of the index
```

Comment:
shall we document that this is used by the rollover max_age math?

Reply:
+1